hack/k8s/helm/nuclio/README.md

```diff
@@ -72,7 +72,7 @@ docker run -d -p 5000:5000 registry:2
 By not providing a registry secret name (`registry.secretName`) nor credentials (`registry.credentials.username` / `registry.credentials.password`), Nuclio will understand credentials are not needed, and not try to load Docker secrets.
 
 ``` sh
-helm install \
+helm install nuclio \
```

Comment:
Looks like all helm commands indentation are wrong, mind you space it correctly?

Reply:
They are spaced correctly, I believe it's the github visualization mucking it up.
Please check in IDE